packages/typeahead/src/react/index.tsx

```diff
@@ -65,6 +75,20 @@ interface TypeaheadFieldStatics {
 type TypeaheadFieldComponent = React.FC<TypeaheadFieldProps> &
   TypeaheadFieldStatics
 
+const defaultFilterFunc = (
+  options: {
+    label: React.ReactText
+    value: React.ReactText
+  }[],
+  inputValue?: string
+) =>
+  options.filter(({ label }: { label: React.ReactText }) =>
+    `${label}`
+      .toLowerCase()
+      // eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
+      .includes((inputValue as string).toLowerCase())
```

Comment:
Function param is nullable string.  Should probably check for null before operating on it and avoid the cast.